src/components/bulkConcepts/component/ActionButtons.jsx

```diff
@@ -59,6 +60,7 @@ export class ActionButtons extends Component {
 
   render() {
     const { disableButton } = this.state;
+    const url_1 = `/orgs/CIEL/sources/CIEL/concepts/${this.props.id}/`;
```

Comment:
A meaningful name for `url_1` could be better.

Reply:
I have fixed that